cristian-bars/challenge/precurse/airlines/airlines-pro.js

```diff
@@ -0,0 +1,184 @@
+let coste = 0;
+let mensaje;
+let nombre;
+let user;
+let contador
+let vuelos = new Array;
+let count;
+
+vuelos = [
+    {id: 0, to: 'Bilbao', from: 'Barcelona', cost: 1600, scale: false},
+    {id: 1, to: 'Barcelona', from: 'Madrid', cost: 1400, scale: false},
+    {id: 2, to: 'Lisboa', from: 'Reus', cost: 2000, scale: false},
+    {id: 3, to: 'Milan', from: 'Oslo', cost: 2500, scale: false},
+    {id: 4, to: 'New York', from: 'Viena', cost: 750, scale: true},
+    {id: 5, to: 'Tokyo', from: 'Sao Pablo', cost: 900, scale: true},
+    {id: 6, to: 'Las Palmas de Gran Canaria', from: 'Menorca', cost: 1200, scale: false},
+    {id: 7, to: 'Tanger', from: 'París', cost: 1100, scale: false},
+    {id: 8, to: 'Los Angeles', from: 'Cuba', cost: 300, scale: false},
+    {id: 9, to: 'Sevilla', from: 'Barcelona', cost: 400, scale: false}
+];
+
+function consulta (){
+    contador = vuelos.length;
+    
+    getName();
+    
+    mensaje = "Bienvenido/a sr/sra " + nombre + "." +
+    " Estos son los vuelos que tenemos para el día de hoy: \n";
+
+    for (let i = 0; i<contador; i++){
```

Comment:
As other reviews it is better to replace such as simple for loops with for..in or for..of loops instead.